frontend/src/slices/app/actions.ts

```diff
@@ -0,0 +1,24 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import { type ValueOf } from 'shared/build/index.js';
+
+import { type NotificationType } from '~/libs/packages/notification/notification.js';
+import { type AsyncThunkConfig } from '~/libs/types/async-thunk-config.type.js';
+
+import { name as sliceName } from './app.slice.js';
+
+type Notification = {
+  message: string;
+  type: ValueOf<typeof NotificationType>;
+};
+
+const notify = createAsyncThunk<unknown, Notification, AsyncThunkConfig>(
+  `${sliceName}/notify`,
+  ({ type, message }, { extra }) => {
+    const { notification } = extra;
+
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-call
```

Comment:
Type it instead of disabling rules
```suggestion

```

Reply:
![image](https://user-images.githubusercontent.com/106602282/226745204-bf8b1529-7db5-453f-bf38-6e7061da1f07.png)
I added this because there is a type error that I cannot solve, how to type it correctly?

Reply:
Have no errors
![image](https://user-images.githubusercontent.com/19575839/226750033-7ae358e3-cc20-4f21-b671-94f80545b29f.png)